# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/pdisk/makefile.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./makefile	2000-05-16 10:02:05.000000000 -0500
+++ ./makefile	2002-12-21 11:57:46.000000000 -0500
@@ -11,7 +11,6 @@
 DOCS= \
 	HISTORY \
 	README \
-	$(MAN_PAGE) \
 	$(MAC_DOC)
 
 SERVER_README = \
@@ -141,6 +140,17 @@
 cvt_pt: $(CVT_OBJECTS)
 	cc -o cvt_pt $(CVT_OBJECTS)
 
+doc_prepare:
+	mkdir doc
+	cp $(DOCS) doc
+
+install: all doc_prepare
+	install -d -m 755 /sbin
+	install -d -m 755 /usr/man/man8
+	install -s pdisk /sbin
+	install -s cvt_pt /sbin
+	install -m 444 pdisk.8 /usr/man/man8
+
 tags:	$(MAC_SOURCE) $(UNIX_SOURCE)
 	ctags $(MAC_SOURCE) $(UNIX_SOURCE)
 
